tests/test_AggregateMSWData.cpp

```diff
@@ -558,6 +558,8 @@ END
 		c.rates.set(o::wat, qw*(float(i)+1.))
                    .set(o::oil, qo*(float(i)+1.))
                    .set(o::gas, qg*(float(i)+1.));
+
+		c.index = 90 + i; // zero-based global index
```

Comment:
Is this a magic number which really should have got a name?

Reply:
Maybe `firstConnectedCell`?

Reply:
I was thinking of the `90`

Reply:
> I was thinking of the integer literal `90`

Yes, and that's the `firstConnectedCell` of this well.  It's the zero-based linear index of cell (1,5,2).

Reply:
OK OK - then I understand; I though the value was something which maybe should be promoted to named constant in the code - now I understand it is a value specific for the test only. Then I am happy to merge a `90` literal.